kontonummer.js

```diff
@@ -203,6 +203,15 @@
             account  : 7,
             control  : 10
         }
```

Comment:
Länsförsäkringar for clearing: 9060-9069 is missing in our config. Should it be added?

Reply:
Isn't Länsförsäkringar `9060-9069` covered by `906[0-9]` part of the regex in:

https://github.com/Advisa/kontonummer.js/blob/3fb909fff52c414deec744b59f595cef315b9e77/kontonummer.js#L180-L181

?

Reply:
Aah sorry you are totally correct. 👍